pushy-benchmark/src/main/java/com/relayrides/pushy/apns/util/ApnsPayloadBuilderBenchmark.java

```diff
@@ -0,0 +1,75 @@
+package com.relayrides.pushy.apns.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.lang3.RandomStringUtils;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+
+@State(Scope.Thread)
+public class ApnsPayloadBuilderBenchmark {
+
+    private ApnsPayloadBuilder apnsPayloadBuilder;
+
+    private static final int MAXIMUM_PAYLOAD_SIZE = 1024 * 8;
+
+    private String shortAsciiMessageBody;
+    private String longAsciiMessageBody;
+    private String shortChineseMessageBody;
+    private String longChineseMessageBody;
+
+    @Setup
+    public void setUp() {
+        this.apnsPayloadBuilder = new ApnsPayloadBuilder();
+
+        this.shortAsciiMessageBody = RandomStringUtils.randomAscii(MAXIMUM_PAYLOAD_SIZE / 8);
```

Comment:
> do you see any differences between re-runs of the test harness?

Actually, no. I would expect more variation if we were using heterogeneous mixes of characters, but everything in the ASCII set takes up one byte in the escaped string, whereas everything in the Chinese character set expands to take up three(?) bytes.

If we had messages that contained a mix of characters with different sizes, we'd definitely want to mutate between runs.